### PR TITLE
pushing up the user model

### PR DESCRIPTION
**Pull request by Mrcaspento**
i deleted some of my files to prevent conflict

### CODE REVIEW

**Comment by mitchmathias** [comment]
conflict is still there, i cant resolve

**Review by boylepeter — CHANGES_REQUESTED** [review]
Matt, we're gonna have to get together in the am and combine our two models. I'm gonna turn this one down for now until we do because I don't want my commit (which is waiting for approval) to overwrite yours.

**Pull request closed without merging**
